resources/tentative/complex-static-html/src/big-dom-generator/html-markup.js

```diff
@@ -0,0 +1,50 @@
+export const ANGULAR_TODO_MVC_HTML_MARKUP = `
+<div class="main-ui" dir="ltr">
```

Comment:
This doesn't match the markup I see on https://speedometer-preview.netlify.app/resources/todomvc/architecture-examples/angular/dist/. 

I was pasting this in the web console to simulate 100 items, although it's not necessary to see the discrepency:

```
var numberOfItemsToAdd = 100;
var newTodo = document.querySelector(".new-todo");
for (let i = 0; i < numberOfItemsToAdd; i++) {
  newTodo.value = `Something to do ${i}`;
  newTodo.dispatchEvent(new Event("input"))
  enter(newTodo)
}


function enter(node, options = undefined) {
  let keyCode = 13;
  let eventOptions = { bubbles: true, cancelable: true, keyCode, key: "Enter", which: keyCode };
  const event = new KeyboardEvent("keyup", eventOptions);
  node.dispatchEvent(event);
}
```

Reply:
We might consider generating the necessary HTML with 100 todo items from the live test contents (starting with something like the above) and pasting it here, compared with needing to keep a duplicate shell and todo generation logic in the script here.

I think that would be an improvement, as it would be easier to keep in sync with changes and I think have less room for error in the codegen script, but I don't feel strongly that it must happen right now.

Reply:
Makes sense. I'd like to get #239 landed to make this easier to compare with